applications/StructuralMechanicsApplication/custom_processes/postprocess_eigenvalues_process.h

```diff
@@ -0,0 +1,221 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+#if !defined(KRATOS_POSTPROCESS_EIGENVALUES_H_INCLUDED )
+#define  KRATOS_POSTPROCESS_EIGENVALUES_H_INCLUDED
+
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "processes/process.h"
+#include "includes/kratos_parameters.h"
+
+#include "custom_io/gid_eigen_io.h"
+
+
+namespace Kratos {
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/// Process to create the animated Eigenvectors
+/** This process takes the results of an Eigenvalue Analysis and creates the 
+ * animated Eigenvectors (Eigenmodes) for GiD using the GidEigenIO, which
+ * is customized for this case
+ * The Input should be the ComputingModelPart! (Otherwise nodal results migth be messed up)
+ * It is of particular importance that all Nodes have the same Dofs!
+ */
+class PostprocessEigenvaluesProcess : public Process
+{
+  public:
+
+    ///@name Type Definitions
+    ///@{
+
+    ///@}
+    ///@name Pointer Definitions
+    /// Pointer definition of PostprocessEigenvaluesProcess
+    KRATOS_CLASS_POINTER_DEFINITION(PostprocessEigenvaluesProcess);
+
+    typedef std::size_t SizeType;
+
+    typedef ModelPart::NodeType::DofsContainerType DofsContainerType;
+    
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    PostprocessEigenvaluesProcess(ModelPart &rModelPart,
+                                  Parameters OutputParameters);
+
+    /// Destructor.
+    ~PostprocessEigenvaluesProcess() = default;
```

Comment:
I don' think you need the explicit mention of the destructor.

Reply:
done